### PR TITLE
Added style to the user interface

### PR DESCRIPTION
**Pull request by frnega**
all the functionality is done and added styling.  used Zeplin as a collaboration tool.

### CODE REVIEW

File: src/styles/book.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  margin: 0;
+}
+
+.book-management {
+  list-style-type: none;
+  padding: 0;
+}
+
+.current-chapter > button {
+  width: 200px;
```
Comment (omarsalem7) [inline src/styles/book.css]: - [ ] Kindly fix your design to match the zipline design, you can fix it by removing `width:200px;` 
> Your design 
image

File: src/styles/book.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  padding: 0;
+  font-family: 'Robotoslab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
```
Comment (omarsalem7) [inline src/styles/book.css]: - [ ] It will be better if you add `background: #fff` to the button to make it match the design in the zipline.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @frnega 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️

- [ ] _Check the comments under the review._
- [ ] Kindly fix the MIT license link and add your own license.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @frnega   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**